hugging_face_classifier_trainer/item.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: v1
+categories:
+- machine-learning
+- model-training
+description: Automatic train, evaluate and predict functions for the ML frameworks
+  - Scikit-Learn, XGBoost and LightGBM.
+doc: ''
+example: hugging_face_classifier_trainer.ipynb
+generationDate: 2022-08-28:17-25
+hidden: false
+icon: ''
+labels:
+  author: davids
+maintainers: []
+marketplaceType: ''
+mlrunVersion: 1.3.0-rc29
+name: hugging_face_classifier_trainer
+platformVersion: 3.5.0
+spec:
+  filename: hugging_face_classifier_trainer.py
+  handler: train
+  image: mlrun/ml-models
+  kind: job
+  requirements:
+  - onnx~=1.10.1
+  - onnxruntime~=1.8.1
+  - optimum
+  - transformers
+  - datasets
+url: ''
+version: 1.1.0
```

@yonishelach is this ok?

---

We don't have a convention to function version yet, I think its better to start with `1.0.0` or even with `0.0.1` if its just an initial form of the function (I guess it will be updated when HuggingFace will be in `mlrun/frameworks`)
```suggestion
version: 0.0.1
```